docs/UserGuide.md

```diff
@@ -1,134 +1,125 @@
-
-#### Listing all persons : `list`
-Shows a list of all persons in the address book.<br>
-Format: `list`
-
-#### Finding all persons containing any keyword in their name: `find`
-Finds persons whose names contain any of the given keywords.<br>
-Format: `find KEYWORD [MORE_KEYWORDS]`
-
-> * The search is case sensitive. e.g `hans` will not match `Hans`
-> * The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-> * Only the name is searched.
-> * Only full words will be matched e.g. `Han` will not match `Hans`
-> * Persons matching at least one keyword will be returned (i.e. `OR` search).
-    e.g. `Hans` will match `Hans Bo`
-
-Examples: 
-* `find John`<br>
-  Returns `John Doe` but not `john`
-* `find Betsy Tim John`<br>
-  Returns Any person having names `Betsy`, `Tim`, or `John`
-
-#### Deleting a person : `delete`
-Deletes the specified person from the address book. Irreversible.<br>
-Format: `delete INDEX`
-
-> Deletes the person at the specified `INDEX`. 
-  The index refers to the index number shown in the most recent listing.<br>
-  The index **must be a positive integer** 1, 2, 3, ...
-
-Examples: 
-* `list`<br>
-  `delete 2`<br>
-  Deletes the 2nd person in the address book.
-* `find Betsy`<br> 
-  `delete 1`<br>
-  Deletes the 1st person in the results of the `find` command.
-
-#### Select a person : `select`
-Selects the person identified by the index number used in the last person listing.<br>
-Format: `select INDEX`
-
-> Selects the person and loads the Google search page the person at the specified `INDEX`. 
-  The index refers to the index number shown in the most recent listing.<br>
-  The index **must be a positive integer** 1, 2, 3, ...
-
-Examples: 
-* `list`<br>
-  `select 2`<br>
-  Selects the 2nd person in the address book.
-* `find Betsy` <br> 
-  `select 1`<br>
-  Selects the 1st person in the results of the `find` command.
-
-#### Clearing all entries : `clear`
-Clears all entries from the address book.<br>
-Format: `clear`  
+
+#### Adding a task: `add`
+Adds a task to the task-list.<br>
+Different Types of tasks (Todo/Event/Deadline) have different command format.<br>
+
+###### Adding a Todo:
+Todos will be rearranged in the Todo-List based on their priority.<br>
+> Format: `add TASK_NAME p/PRIORITY`<br>
+> Example: `add Assignment 3 p/1`
+
+###### Adding an Event:
+> Format: `add TASK_NAME d/DATE s/START_TIME e/END_TIME`<br>
+> Example: `add Time's birthday party d/25-12-2016 s/1400 e/1600`
+
+###### Adding a Deadline:
+> Format: `add TASK_NAME d/DATE e/END_TIME`<br>
+> Example: `add CS2103 v0.2 d/25-12-2016 e/1600`
+
+#### Editing a task: `edit`
+Edits information of the task in the task-list.<br>
+Different Types of tasks (Todo/Event/Deadline) have different command format.<br>
+
+###### Editing a Todo:
+Todos will be rearranged in the Todo-List based on their priority.<br>
+> Format: `edit TASK_TYPE INDEX_NUMBER n/TASK_NAME p/PRIORITY`<br>
```

Comment:
- why is it necessary to indicate the task type?
- why should i change the name and priority at the same time? what if i want to update only one?
- the same comment holds for other edit commands as well.